text/0000-const-ub.md

```diff
@@ -0,0 +1,111 @@
+- Feature Name: `const_ub`
+- Start Date: 2020-10-10
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Define UB during const evaluation to lead to an unspecified result or hard error for the affected CTFE query, but not otherwise infect the compilation process.
```

Comment:
Should I make it explicit that running into internal compiler errors is *not* an acceptable reaction to CTFE UB, i.e., if there are errors, they must be "proper diagnostics", not "exceptions"? I am not entirely sure how to word this though.

Reply:
That's unnecessary imo, as the compiler team already treats all ICEs on stable as bugs and requires the minimal fix to be to at least hide the ICE via a proper error or by delaying it.